src/main/java/io/coala/jetbrains/utils/CodeAnalysisLog.java

```diff
@@ -0,0 +1,69 @@
+package io.coala.jetbrains.utils;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.TimeZone;
+
+public class CodeAnalysisLog {
```

Comment:
This class represents a line in the log, right? If so, please name is appropriately.

Reply:
Each instance represents one log block in the JSON having severity, message, and timestamp of when it was generated. I kept the name similar to `CodeAnalysisIssue` (each instance represents one issue) since both are similar in nature.

Reply:
What does `issue` here refer to? You are not using this to get lint results, are you?